python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
     fits:tunit:
     votable:arraysize: "*"
     ivoa:ucd: meta.id;instr.filter;meta.main
+    tap:principal: 1
+    tap:column_index: 112
   - name: refExtendedness
     "@id": "#Object.refExtendedness"
     datatype: double
     description: Set to 1 for extended sources, 0 for point sources. Reference band.
     fits:tunit:
+    tap:principal: 1
+    tap:column_index: 113
   - name: refFwhm
     "@id": "#Object.refFwhm"
     datatype: double
     description: Estimated FWHM on the reference band at source center assuming a Gaussian profile
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 114
   - name: shape_flag
     "@id": "#Object.shape_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Reference band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 115
   - name: shape_xx
     "@id": "#Object.shape_xx"
     datatype: double
     description: Hirata-Seljak-Mandelbaum (HSM) moments. Reference band.
     fits:tunit: pixel**2
+    tap:principal: 0
```

Comment:
Calling these moments the "HSM" ones is probably more confusing than helpful (affects many other similar columns).  The HSM paper is about a method ("regaussianization") for going from Gaussian-weighted second moments (which is what these are) to deconvolved shear estimators useful for weak lensing.  We happen to use a heavily modified (first by GalSim, then by us) version of some code from HSM to compute those Gaussian-weighted second moments, but that's a very standard algorithm and nobody outside of DM associates it with "HSM" (and we, DM, probably shouldn't, either).  We should only use "HSM" in our description of the "regauss" shape columns.